libs-haskell/da-hs-base/src/DA/Service/Logger/Impl/README.md

```diff
@@ -0,0 +1,12 @@
+# Getting the data out of GCP
+
+You can get the data out of GCP, provided you are a digital asset employee with sufficient permissions. First ask for permissions on "Logs Viewer" and "BigQuery User" on the `stackdriver-customer-telemetry` project from an administrator.
+
+Then log into [Google Cloud console](https://console.cloud.google.com/). To view live logging click the hamburger icon and select Logging -> Log viewer from the side panel. In the far left dropdown menu below the filter select "Global" for the logs to view. This will show you all the logs from the last 30 days.
```

Comment:
I can't see a hamburger icon and I honestly don't understand what you're referring to.
I had to select a project as well.